multi-node/aws/pkg/cluster/stack_template.go

```diff
@@ -223,6 +224,7 @@ func StackTemplateBody(defaultArtifactURL string) (string, error) {
 				map[string]interface{}{"IpProtocol": sgProtoUDP, "FromPort": 0, "ToPort": sgPortMax, "CidrIp": sgAllIPs},
 			},
 			"SecurityGroupIngress": []map[string]interface{}{
+				map[string]interface{}{"IpProtocol": sgProtoICMP, "FromPort": 3, "ToPort": -1, "CidrIp": sgAllIPs},
```

Comment:
Fair argument, I'm happy supporting this. I'm not quite clear on what the appropriate FromPort/ToPort values would be, though. My immediate thought is 3/3, but you've proposed 3/-1. What am I missing?